src/components/landing-page/call-to-action/Cards.jsx

```
@@ -1,87 +1,109 @@
+import React, { memo } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)